app/containers/wallet/WalletAddPage.js

```diff
@@ -99,4 +105,8 @@ export default class WalletAddPage extends Component<Props> {
     return this.props.stores.substores[environment.API].wallets;
   }
 
+  _getTrezorConnectStore() {
```

Comment:
missing return type

Reply:
i tried but not sure how to do it,

```
_getWalletsStore() {
    return this.props.stores.substores[environment.API].wallets;
  }
```
is also not using type, so don't know how to figure it out.

any help?

Reply:
import class and use it as type

Reply:
DONE